zcash_primitives/CHANGELOG.md

```diff
@@ -22,11 +23,18 @@ and this library adheres to Rust's notion of
   - `zip317::FeeRule::non_standard` has been moved behind the `non-standard-fees`
     feature flag. Using a non-standard fee may result in a transaction that cannot
     be mined on the current `Zcash` network.
+- Refactored `AccountBalance` to use `Balance` for transparent funds (issue #1411).
+  `AccountBalance` now has an `unshielded_balance()` function that returns `Balance`. 
+  This does not currently distinguish between pending change and non-change; the 
+  pending value is all counted as non-change. Ability to distinguish such funds is 
+  tracked by issue #1592
```

Comment:
This part is not part of the `WalletRead` API; it's a bug/limitation in `zcash_client_sqlite`'s implementation of that API for `WalletDb`. All other changes to this file should instead be to `zcash_client_backend/CHANGELOG.md`.

Reply:
Oh, that's right. Thank you for pointing it out.